src/reducers/__test__/reducers.spec.js

```diff
@@ -0,0 +1,98 @@
+// import { combineReducers } from 'redux';
```

Comment:
Hey @cvjude. Is there a reason why these lines were commented out? Also if they are not useful, I'd suggest we take them out entirely.